website/docs/tutorials/adding-more-runtimes/prisma-no-sugar.py

```diff
@@ -1,28 +1,30 @@
-# skip:start
-from typegraph import TypeGraph, effects, t
-from typegraph.providers.prisma.runtimes.prisma import PrismaOperationMat, PrismaRuntime
-
-with TypeGraph("prisma-no-sugar") as g:
-    db = PrismaRuntime("database", "POSTGRES_CONN")
-    message = t.struct({})
-    # skip:end
-    t.func(
-        t.struct(
-            {
-                "data": t.struct(
-                    {
-                        # notice to absence of `id` as automatically generated
-                        "title": t.string(),
-                        "body": t.string(),
-                    }
-                )
-            }
-        ),
-        t.array(message),
-        PrismaOperationMat(
-            db,
-            "Message",
-            "createOne",
-            effect=effects.create(),
-        ),
-    )
+# # skip:start
+# from typegraph_next import typegraph, effects, t, Graph
+# from typegraph_next.providers.prisma import PrismaRuntime
+#
+#
+# @typegraph()
+# def prisma_no_sugar(g: Graph):
```

Comment:
any issue with that one?

Reply:
We do not expose materializers directly to the user anymore.